frontend/src/app/modules/common/hide-section/add-section-dropdown/add-section-dropdown.component.ts

```diff
@@ -30,11 +30,12 @@ import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 import {Component, ElementRef, OnDestroy, OnInit, ViewChild} from "@angular/core";
 import {untilComponentDestroyed} from "ng2-rx-componentdestroyed";
 import {HideSectionDefinition, HideSectionService} from "core-app/modules/common/hide-section/hide-section.service";
-import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 import {AngularTrackingHelpers} from "core-components/angular/tracking-functions";
 
+export const addSectionDropdownSelector = 'add-section-dropdown';
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3b1eae5a64159f8a044579a25870c772186dbd3f/frontend/src/app/modules/common/hide-section/add-section-dropdown/add-section-dropdown.component.ts#L36"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>